extensions/applicationinsights-debugplugin-js/src/DebugPlugin.ts

```diff
@@ -0,0 +1,295 @@
+/**
+ * DebugPlugin.ts
```

Comment:
Please copy the copyright header from the other components as the build process removes and replaces it with the specific version number included.

Reply:
for this you mean the:

```ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

header, correct?